lib/node_modules/@stdlib/assert/is-well-formed-string/examples/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable no-new-wrappers */
+
+'use strict';
+
+var isWellFormedString = require( './../lib' );
+
+console.log( isWellFormedString( '' ) );
+// => true
+
+console.log( isWellFormedString( new String( '' ) ) );
+// => false
```

Comment:
Shouldn't this be `true`? Am confused why linting didn't pick on this...

```suggestion
// => true
```

Reply:
Yes, it should be true.

Reply:
Thanks for confirming!